script.service.ltvpl/resources/data/dataFileLocation.py

```diff
@@ -0,0 +1,2 @@
+#
+dataFilePath = r"D:\My Documents\GitHub\LiveTV_PlayList\script.service.ltvpl\resources\data\LTVPL.pkl"
```

Comment:
hardcoded path, this won't work

Reply:
This absolute path is replaced at runtime by a relative path when the code runs under the Kodi environment. This code also runs under Windows which is why you see that absolute path. It does work on all my systems running Kodi ( Raspberry Pi & Windows).